projects/plugins/boost/app/assets/src/js/features/super-cache-info/super-cache-info.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import { DataSyncProvider, useDataSync } from '@automattic/jetpack-react-data-sync-client';
+import { __, sprintf } from '@wordpress/i18n';
+import { Notice, Button } from '@automattic/jetpack-components';
+import {
+	measureSuperCacheSaving,
+	isSuperCachePluginActive,
+	isSuperCacheEnabled,
+} from './lib/utils/measure';
+import { z } from 'zod';
+
+const SuperCacheInfo = () => {
+	const [ state, setState ] = useState( { status: 'idle' } ); // idle, testing, error, complete
+
+	const [ { data }, { mutate } ] = useDataSync(
+		'jetpack_boost_ds',
+		'super_cache_notice_disabled',
+		z.boolean()
+	);
+
+	const isNoticeDismissed = data;
+	const setNoticeDismissed = mutate;
```

Comment:
In JavaScript, you can destructure variables under any name you like. So you could simplify this like so: 

```suggestion
	const [ { data: isNoticeDismissed }, { mutate: setNoticeDismissed } ] = useDataSync(
		'jetpack_boost_ds',
		'super_cache_notice_disabled',
		z.boolean()
	);
```